db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/amazon_check_pre_retirement.rb

```diff
@@ -5,22 +5,24 @@
 
 # Get vm from root object
 vm = $evm.root['vm']
+ems = vm.ext_management_system
```

Comment:
@billfitzgerald0120 @tinaafitz 
Can the vm be nil, isn't that an error case. We might have a vm not connected to an ems but we need at the minimum a vm to work. If the vm is nil should we throw an ABORT